node/tests/ui_gateway_test.rs

```diff
@@ -48,19 +48,11 @@ fn request_financial_information_integration() {
         true,
     );
     node.wait_for_log("UIGateway bound", Some(5000));
-    let financials_request = UiFinancialsRequest {
-        payable_minimum_amount: 0,
-        payable_maximum_age: 1_000_000_000_000,
-        receivable_minimum_amount: 0,
-        receivable_maximum_age: 1_000_000_000_000,
-    };
+    let financials_request = UiFinancialsRequest {};
```

Comment:
Probably only one of these tests is necessary; but the test that's chosen to remain should probably be one that makes it possible to assert on the response. Asserting on the financials response might be as easy as making sure all the numbers are zero; asserting on the descriptor response would involve scraping the log ("wait_for_log") to see what the expected value should be. Or perhaps a third message with an even more deterministic response could be chosen.

The objective of these tests seems to be that the UIGateway is exercised both inwardly and outwardly in some way; exactly what message is used to do so is probably not important.